src/content/guides/line-items.mdx

```diff
@@ -56,3 +56,32 @@ nav:
       }
     ]
     ```
+
+## Determining Paid For Line Items for Partial Payments
```

Comment:
I believe this is independant of partial payments.
Maybe there should just be some context on ways this is useful, like a partial payment.

Reply:
I removed " for Partial Payments" and added a bit more context/reworded the paragraph below. Could you give it another read please? :)